relay/kusama/Cargo.toml

```diff
@@ -1,286 +1,321 @@
 [package]
-name = "kusama-runtime"
-build = "build.rs"
-version.workspace = true
 authors.workspace = true
+build = "build.rs"
+description = "Runtime of the Kusama Network"
 edition.workspace = true
+license.workspace = true
+name = "staging-kusama-runtime"
```

Comment:
Does this still need to be called `staging-kusama-runtime` opposed to just `kusama-runtime`?

Reply:
Yep since we dont own https://crates.io/crates/kusama-runtime 🙈